tests/__init__.py

```diff
@@ -111,3 +112,96 @@ def generate_test_case(num_axes, num_ranges, max_value):
     expected_output = merge_ned_ranges(input_ranges)
 
     return input_ranges, expected_output
+
+
+def generate_xml(has_macro=True, has_label=True, root_tag="OME", num_images=1):
+    """Generate synthetic XML strings with options to include 'macro' and 'label' images."""
+
+    # Create the root element
+    ome = ET.Element(
+        root_tag,
+        {
+            "xmlns": "http://www.openmicroscopy.org/Schemas/OME/2016-06",
+            "xmlns:xsi": "http://www.w3.org/2001/XMLSchema-instance",
+            "Creator": "tifffile.py 2023.7.4",
+            "UUID": "urn:uuid:40348664-c1f8-11ee-a19b-58112295faaf",
+            "xsi:schemaLocation": "http://www.openmicroscopy.org/Schemas/OME/2016-06 http://www.openmicroscopy.org/Schemas/OME/2016-06/ome.xsd",
```

Comment:
Are these liable to change? Would we want to pop them or other specification out into a toml config or something similar?

Reply:
I fixed it using some other `asserts` making it idempotent so now versions and headers should not make difference. Thanks!